docs/websites/README.md

```diff
@@ -1,11 +1,8 @@
 # How to Set Up a Local Web Server for the Keyman Web Pages
 
-Currently, most of the Keyman websites are running via IIS. Refer to the Keyman [wiki](https://github.com/keymanapp/keyman/wiki/How-to-set-up-a-local-web-server-for-the-Keyman-web-pages) for setting that up on Windows 10.
+Currently, most of the Keyman websites (*.keyman.com, *.keyman-staging.com) are running PHP 7.4 via Apache.
 
-To make IIS redirects compatible with the Docker sites below, see
-https://github.com/keymanapp/keyman.com/issues/337#issuecomment-1336339387
-
-As the websites get migrated to Apache via Docker, follow the installation steps below:
+For legacy configuration of IIS on Windows 10, refer to the deprecated Keyman [wiki](https://github.com/keymanapp/keyman/wiki/%5BDeprecated%5D-How-to-set-up-a-local-IIS-web-server-for-the-Keyman-web-pages).
```

Comment:
This isn't even worth mentioning any more because we don't have the web.config files in a visible branch on the sites, so they just won't work. We can just delete.